dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/services/EmailSender.kt

```diff
@@ -1,48 +1,49 @@
-package org.dataland.datalandbackend.services
+package org.dataland.datalandcommunitymanager.services
 
 import com.mailjet.client.ClientOptions
 import com.mailjet.client.MailjetClient
 import com.mailjet.client.errors.MailjetException
 import com.mailjet.client.transactional.SendEmailsRequest
 import com.mailjet.client.transactional.TransactionalEmail
-import org.dataland.datalandbackend.model.email.Email
-import org.dataland.datalandbackend.model.email.email
+import org.dataland.datalandcommunitymanager.model.email.Email
+import org.dataland.datalandcommunitymanager.model.email.integrateEmailIntoTransactionalEmailBuilder
 import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Value
 import org.springframework.stereotype.Component
 
 /**
  * A class that manages sending emails
  */
 @Component
 class EmailSender(
-    mailServerUrl: String = "https://api.eu.mailjet.com",
+    @Value("\${mailjet.api.url}") private val mailjetApiUrl: String,
```

Comment:
What does the change do?

Reply:
The mailjet api url is now set in the application properties of the spring service, and only referenced here.
That is cleaner than hardcoding it here as it was before.

Reply:
ok